app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/view_lloyd_george_not_bsol_workflow.cy.js

```diff
@@ -261,9 +261,7 @@ describe('GP Workflow: View Lloyd George record', () => {
 
                     // Assert contents of page when downloading
                     cy.contains('Downloading documents').should('be.visible');
-                    cy.getByTestId(
-                        'download-file-header-' + viewLloydGeorgePayload.number_of_files + '-files',
-                    ).should('be.visible');
+                    cy.getByTestId('download-file-header-0-files').should('be.visible');
```

Comment:
Is it correct to expect test id "download-file-header-0-files"? 
From the code that generates this test id, it seems that `download-file-header-0-files` indicates the text showing "Preparing download for 0 files", which is the issue that we should probably fix.
When I actually run this cypress test, the screen shows "download for 0 files" as well
![image](https://github.com/nhsconnect/national-document-repository/assets/127404525/4534871e-a302-48f1-8c3c-26d193cb9748)

Reply:
This is because the non-bsol route doesn't call the get documents list lambda. So we don't have the search results at any point, which is what we use for the documents number.
Before the UI was using the number that is returns from the stitch lambda about the number of files.
So to fix this we would need to pass in again the number of file from the stitch lambda. 
The reason I change this to 0 was that the only time I found the file number is being used in the non-Bsol workflow was in the test-id. But if we also use it for this screen I'll change it back.

Reply:
Done